### DISCUSSION TITLE
Activity column in FBS files

### DISCUSSION THREAD

**Discussion by wadedavis13** [USEPA/flowsa#361] [Q&A]
Is there a way to modify yaml method file to retain the "ActivityProducedBy" column in FlowBySector output files?
I'm finding this feature would be pretty helpful in our FlowBySector-to-SAGE mapping exercise.
Thanks! 
@WesIngwersen @bl-young

**Comment by bl-young** [comment]
No you can't keep the Activity columns in the FBS output files. However in v2.0 (recursive-refac branch), the metadata field of the FBS will identify the primary data source **AND** the name of the activity set. So for [GHG_national](https://github.com/USEPA/flowsa/blob/recursive-refac/flowsa/methods/flowbysectormethods/GHG_national_m1_common.yaml#L68-L85) you'd get metadata fields such as:
- `EPA_GHGI_T_2_1.direct`
- `EPA_GHGI_T_2_1.liming`
- `EPA_GHGI_T_2_1.electricity_transmission`
- etc.

In that way the FBS is not quite so aggregated as before to enable better review of the particular activities behind each record. If there are specific activities you want to call out you can make them a new named activity set, duplicating the attribution approach.
